### ISSUE TITLE
[rad-subscribe] CLI tool for processing websocket events

### ISSUE THREAD

**Issue by Ryanmtate** [radicle-dev/radicle-client-tools#16]
`rad-subscribe ws://org-node -- ./my-script` -> calls `./my-script <proj> <peer> <ref>`

To be used for mirroring, CI, or other process. Add a separate PR with a mirror example.

**Comment by Ryanmtate** [comment]
Have some work started on this, will finish after https://github.com/radicle-dev/radicle-client-services/issues/70 is able to be unblocked.
